### PR TITLE
Update encodings and documentation

### PR DESCRIPTION
**Pull request by cthoyt**
- Add `# -*- coding: utf-8 -*-` to first line of all python files
- Update documentation for poor suckers still on py36

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -16,7 +16,9 @@ easy_config |python_versions| |license| |develop_build| |develop_coverage|
 Example
 -------
 
-Here is a full working example of using easy_config. First, write your configuration class::
+Here is a full working example of using easy_config. First, write your configuration class:
+
+.. code-block:: python
```
Comment (scolby33) [inline README.rst:21]: RST might just be the worst.

Comment (cthoyt) [reply README.rst:21]: Why did you have double colons?

File: setup.py [hunk 2]
```diff
@@ -1,3 +1,5 @@
+# -*- coding: utf-8 -*-
```
Comment (scolby33) [inline setup.py:1]: The literal worst

Comment (cthoyt) [reply setup.py:1]: NO UR THE WORST ACCEPT MY PR

File: README.rst (hunk 1 above)
Comment (scolby33) [reply README.rst:21]: That creates a code block.

File: setup.py [hunk 3]
```diff
@@ -1,3 +1,5 @@
...
+
```
Comment (cthoyt) [inline setup.py:2]: This was the worst

File: README.rst (hunk 1 above)
Comment (cthoyt) [reply README.rst:21]: No shit

Comment (cthoyt) [reply README.rst:21]: Wow

Comment (scolby33) [reply README.rst:21]: Yep, because RST is the worst.

**Pull request merged**